front/js/components/categoryFilters.es6

```diff
@@ -11,7 +11,8 @@
 
   const config = {
     hiddenFilterGroupIds: getHiddenFilterGroupIds(),
-    filterGroup: 'data-tag-group',
+    filterGroup: 'tag-group',
+    isCollapseInversed: $('.js-tags-list').data('default-collapse'),
```

Comment:
this might be more clear:
`collapseByDefault: $('.js-tags-list').data('collapse-by-default'),`, but up to you